### PR TITLE
Error message fix

### PR DESCRIPTION
**Pull request by gbidsilva**
When a parsing error occurred during the CSV data parsing, the exception message contains duplicate Exception class name.

Relates to: https://issues.apache.org/jira/browse/CSV-309

### CODE REVIEW

File: src/main/java/org/apache/commons/csv/CSVParser.java [hunk 1]
```diff
@@ -147,7 +147,7 @@ private CSVRecord getNextRecord() {
             try {
                 return CSVParser.this.nextRecord();
             } catch (final IOException e) {
-                throw new UncheckedIOException(e.getClass().getSimpleName() + " reading next record: " + e.toString(), e);
+                throw new UncheckedIOException("Error in reading next record: " + e.toString(), e);
```
Comment (michael-o) [inline src/main/java/org/apache/commons/csv/CSVParser.java:150]: Why not just `throw new UncheckedIOException("Error in reading next record", e);`?  No duplication. Plain and simple.

Comment (garydgregory) [inline src/main/java/org/apache/commons/csv/CSVParser.java:150]: "Error" has a specific meaning in Java, and we did _not_ catch a `java.lang.Error` here. I'll adjust.

Comment (gbidsilva) [reply src/main/java/org/apache/commons/csv/CSVParser.java:150]: Might be the intension was to provide the sense of error in the message itself.

Comment (gbidsilva) [reply src/main/java/org/apache/commons/csv/CSVParser.java:150]: I feel "Exception reading next record" would be more meaningful.

Comment (michael-o) [reply src/main/java/org/apache/commons/csv/CSVParser.java]: Why not just: "Failed to read next record"?

**Pull request merged**